apps/services/endorsements/api/src/app/modules/endorsement/interceptors/endorsement.mask.ts

```diff
@@ -3,18 +3,21 @@ import { Endorsement } from '../models/endorsement.model'
 export function maskEndorsement(
   endorsement: Endorsement,
   isListOwner: boolean,
+  isAdmin: boolean,
 ): Endorsement {
   // endorsement data display rules
-  // always mask out nationalId for all
-  endorsement.endorser = 'xxxxxx-xxxx'
-  if (!isListOwner) {
-    // only owner sees locality
-    endorsement.meta.locality = ''
-    // public and admins see name if endorser sets it to be shown
+  // always mask out nationalId for all user groups
+
+  if (isListOwner) {
```

Comment:
Maybe just have this outside any if statements if its always masked?
 endorsement.endorser = 'xxxxxx-xxxx'

Reply:
Im doing it this way for better readability - the old way had higher cognitive load imo